roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -25,7 +25,8 @@
     signer_serial: "{{ openshift_master_config_dir }}/ca.serial.txt"
     hostnames:
     - "{{ docker_registry_service_ip.results.clusterip }}"
-    - docker-registry.default.svc.cluster.local
+    - docker-registry.default.svc
+    - "docker-registry.default.svc.{{ openshift.common.dns_domain }}"
```

Comment:
Should we try to keep these openshift_facts vars out of the internals of the role as we've done with other facts for registry?